src/isPalindrome.js

```diff
@@ -0,0 +1,13 @@
+export default function isPalindrome(inputString){
+
+  inputString = inputString.toLowerCase();
+  inputString = inputString.replace(/[^\w]|_/g, "");
+  var inputArray = inputString.split('');
+  inputArray.reverse();
+  var reverseString = inputArray.join('');
+  if (reverseString === inputString) {
```

Comment:
When I see the pattern:
```js
if( boolean_expression ) {
  return true
} else {
  return false
}
```
I replace it with the more concise:
```js
return boolean_expression
```
In this case, you can simplify your code to be:
```js
return input === reverse
```